core/common/src/main/java/alluxio/PropertyDocGeneration.java

```diff
@@ -0,0 +1,138 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.annotation.concurrent.ThreadSafe;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.lang.reflect.Field;
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * A utility to generate property keys to csv files.
+ */
+
+@ThreadSafe
+public final class PropertyDocGeneration {
+  private static final Logger LOG = LoggerFactory.getLogger(PropertyDocGeneration.class);
+
+  private PropertyDocGeneration() {
+  }
+
+  /**
+   * Write property key to csv files.
```

Comment:
NIT: "Write" -> "Writes", this is an Alluxio convention to use present tense for javadoc